dashboard/app/controllers/api_controller.rb

```diff
@@ -101,12 +102,18 @@ def user_progress_for_stage
           timestamp: last_activity.updated_at.to_datetime.to_milliseconds,
           source: level_source
         }
-
       end
       response[:disableSocialShare] = current_user.under_13?
       response[:disablePostMilestone] =
-        !Gatekeeper.allows('postMilestone', where: {script_name: script.name}, default: true)
+          !Gatekeeper.allows('postMilestone', where: {script_name: script.name}, default: true)
     end
+
+    slog(tag: 'activity_start',
+         script_level_id: script_level.id,
+         level_id: level.id,
+         user_agent: request.user_agent,
+         locale: locale) if level.finishable?
```

Comment:
We don't currently query this API any time other than on starting a level, correct? Should we make a note somewhere noting this added requirement, or would extra activity-start events not be a problem if we do ever start querying this API more frequently?